test/functional/test_framework/test_node.py

```diff
@@ -167,6 +177,9 @@ def start(self, extra_args=None, *, stdout=None, stderr=None, **kwargs):
         self.running = True
         self.log.debug("bitcoind started, waiting for RPC to come up")
 
+        if self.start_perf:
+            self._start_perf("")
```

Comment:
That will  overwrite previous perf results.

Reply:
Fixed and tested by running `./test/functional/p2p_disconnect_ban.py --perf` (which restarts node1):
```sh
 $ ./test/functional/p2p_disconnect_ban.py --perf
2018-11-09T18:41:23.959000Z TestFramework (INFO): Initializing test directory /tmp/testk5t2fuyc
[...]
2018-11-09T18:41:25.250000Z TestFramework (INFO): setban: test persistence across node restart
2018-11-09T18:41:25.616000Z TestFramework.node1 (INFO): Wrote perf output to '/tmp/testk5t2fuyc/node1/perf.data'
[...]
2018-11-09T18:41:26.438000Z TestFramework (INFO): Stopping nodes
2018-11-09T18:41:26.756000Z TestFramework.node0 (INFO): Wrote perf output to '/tmp/testk5t2fuyc/node0/perf.data'
2018-11-09T18:41:27.040000Z TestFramework.node1 (INFO): Wrote perf output to '/tmp/testk5t2fuyc/node1/1.perf.data'
2018-11-09T18:41:27.041000Z TestFramework (WARNING): Not cleaning up dir /tmp/testk5t2fuyc due to perf data
2018-11-09T18:41:27.041000Z TestFramework (INFO): Tests successful
```